Godeps/_workspace/src/github.com/miekg/dns

```diff
@@ -0,0 +1 @@
+Subproject commit 5d001d020961ae1c184f9f8152fdc73810481677
```

Comment:
Looks like you created a subproject. This does not work. Can you remove the subproject from the commit and then do:
> go get miekg/dns
godep save

in order to add miekg/dns to the godep tree.